WordPress/gutenberg
# Show most frequently used blocks next to inserter

## Description
Instead of hardcoded paragraph and image blocks, this shows
the three most frequently used blocks, based on stored blockUsage

The algorithm for working out the blocks to display is just "most frequently used." I think that's fine for now, but we probably want to think about future refinements that "age" the usage too, in the future.

## How to test
Hover over the inserter icon in the main editing section, and your three most used blocks should appear next to it. Click on them to insert them.

Clear your local storage, and paragraph and image should be displayed to start with.
